pkg/executables/kubectl.go

```diff
@@ -825,3 +825,116 @@ func (k *Kubectl) GetBundles(ctx context.Context, kubeconfigFile, name, namespac
 
 	return response, nil
 }
+
+func (k *Kubectl) PatchTolerations(ctx context.Context, taints []corev1.Taint, resource string, name string, kubeconfigFile string, namespace string, path string) error {
```

Comment:
I would suggest creating a wrapper like `PatchDaemonSetTolerations` and omit the resource argument there
Similar to what we do for patch image
And you might be able to do the same for the path